randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -963,19 +963,31 @@ Extra - minimap_data: [{'x': 70, 'y': 44}, {'x': 70, 'y': 45}, {'x': 70, 'y': 46
   * Extra - instance_id: 134710
   > Door to Bullet Hell Room
       Trivial
+  > EMP Ball
+      Any of the following:
+          Morph Ball and Morph Glide (Advanced)
+          Any of the following:
+              # Shoot the EMP from the door
+              Super Missiles
+              # RNG dependant
+              Missiles and Knowledge (Expert)
 
 > Door to Bullet Hell Room; Heals? False
   * Layers: default
   * Distribution Center Bullet Hell Room Access EMP Door to Bullet Hell Room/Door to Bullet Hell Room Access
   * Extra - instance_id: 134711
-  > Door to Distribution Facility Intersection
+  > Center
       Any of the following:
```

Comment:
Just realized, you can probably do this with HJ+CBJ, no?

Reply:
Tested and no, but you can do from EMP -> Bullet Hell Room with HJ+CBJ
![image](https://github.com/randovania/randovania/assets/125271738/2ec8cac7-13d7-4374-9e34-52b0edf51bde)

Reply:
Sorry, I forgot to include walljump <-<
Meant something like this
![grafik](https://github.com/randovania/randovania/assets/38186597/f77a0fa0-9a15-4bb6-893d-5b797453b418)

Reply:
Okay this can be done (from both sides). Rating it at WJ (Int) and CBJ (Adv)? It's a bit tighter getting around the wall so I think it needs the bump in difficulty.

Reply:
yup seems fine to me